### DISCUSSION TITLE
Is there any way to get the dispatch key

### DISCUSSION THREAD

**Discussion by ss22219** [Melledy/Grasscutter#1235] [Q&A] [answered]
正式环境中使用 /keys/dispatchKey.bin 加密数据，我想用来做圣遗物数据分析，请问有什么办法获取到这个文件吗

In the rel version, /keys/dispatchKey.bin is used to encrypt data. I want to use it for data analysis of Artifact. Is there any way to get this file?

**Comment by ss22219** [comment]
像这个项目
https://github.com/HolographicHat/YaeAchievement

**Comment by zhaodice** [comment, marked as answer]
你可能需要这个
https://github.com/Grasscutters/Grasscutter/tree/development/src/main/resources/keys
